ui/DesignSystem/Component/RemoteHelperHint.svelte

```diff
@@ -0,0 +1,53 @@
+<script>
+  import { createEventDispatcher } from "svelte";
+
+  import { Icon } from "../../DesignSystem/Primitive";
+  import Copyable from "./Copyable.svelte";
+
+  const dispatch = createEventDispatcher();
+
+  export let style = null;
+  export let hidden = false;
```

Comment:
Instead of having a prop the calling side can decide to wrap the component in an if block and therefore not even invoke it.

Reply:
Good point!
https://github.com/radicle-dev/radicle-upstream/pull/769/commits/a12c2b846a35b815ad8bec0c945a1d1e9962eae0